docs/userguide/networking/get-started-overlay.md

```diff
@@ -21,6 +21,8 @@ some pre-existing conditions before you can create one. These conditions are:
 * A cluster of hosts with connectivity to the key-value store.
 * A properly configured Engine `daemon` on each host in the cluster.
 
+Though Docker Machine and Swarm are not mandatory to experience Docker multi-host-networking,
```

Comment:
Though Docker Machine and Swarm are not mandatory to experience Docker
multi-host-networking, this example uses them to illustrate how they are
integrated.